CONTRIBUTING.md

```diff
@@ -96,10 +69,10 @@ Detailed description
 
 The `Area of changes` is related either to functional domain (i.e. Build, Release) or feature domain (i.e. Explore, Plugins, BarGaugePanel).
```

Comment:
Yea. We dont enforce the commit style. Since we squash PR's it's impossible to enforce it. Sometimes people forget to follow all commit style guides when merging. I dont think that's a big problem. PR titles can be updated afterward if we see they dont follow the convention.,